students/sangwoong/westagram/users/view.py

```diff
@@ -45,16 +49,21 @@ class SigninView(View):
     def post(self, request):
         try:
             data = json.loads(request.body)
+            password=data['password']
+            user=User.objects.get(email=data['email'])
 
             if data['email']=="" or data['password']=="":
                 return JsonResponse({"MESSAGE":"NOT_FOUND"},status=404)
             
             if not User.objects.filter(email=data['email']).exists():
                 return JsonResponse({"MESSAGE" : "INVALID_USER"}, status=401)
                 
-            if User.objects.get(email=data['email']).password !=data['password']:
-                return JsonResponse({"MESSAGE" : "INVALID_USER"}, status=401)
+#            if User.objects.get(email=data['email']).password !=data['password']:
+#                return JsonResponse({"MESSAGE" : "INVALID_USER"}, status=401)
+            
+            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
+                return JsonResponse({'MESSAGE':'INVALID_USER_bcrypt'}, status=401)
```

Comment:
```suggestion
            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
                return JsonResponse({'MESSAGE':'INVALID_USER'}, status=401)
```
어떤 라이브러리를 썼는지 굳이 알려줄 필요는 없습니다.